### PR TITLE
Issue#217 get signature

### PR DESCRIPTION
**Pull request by debian-sh**
It's example for returned response on request to `/api/v1/description`

```json
[
    {
        "name": "user",
        "human_readable_name": "User",
        "human_readable_description": "This resource type was created for handling users, it allows users to set up accounts. After creating an account, the user will be able to login to the your application.",
        "body": {
            "username": {
                "name": "username",
                "type_name": "text",
                "human_readable_name": {},
                "type": {
                    "params": {}
                },
                "required": true,
                "derived": false
            },
            "email": {
                "name": "email",
                "type_name": "email",
                "human_readable_name": {},
                "type": {
                    "params": {}
                },
                "required": false,
                "derived": false
            },
            "password": {
                "name": "password",
                "type_name": "text",
                "human_readable_name": {},
                "type": {
                    "params": {}
                },
                "required": true,
                "derived": false
            },
            "status": {
                "name": "status",
                "type_name": "text",
                "human_readable_name": {},
                "type": {
                    "params": {}
                },
                "required": false,
                "derived": false
            }
        }
    }
]
```

### CODE REVIEW

**Comment by debian-sh** [comment]
```json
[
	{
	    "name": "user",
	    "human_readable_name": "User",
	    "human_readable_description": "This resource type was created for handling users, it allows users to set up accounts. After creating an account, the user will be able to login to the your application.",
	    "body": {
	        "0": {
	            "name": "username",
	            "type": "text",
	            "required": true,
	            "params": {}
	        },
	        "1": {
	            "name": "email",
	            "type": "email",
	            "required": false,
	            "params": {}
	        },
	        "2": {
	            "name": "password",
	            "type": "text",
	            "required": true,
	            "params": {}
	        },
	        "3": {
	            "name": "status",
	            "type": "text",
	            "required": false,
	            "params": {}
	        }
	    }
	}
]
```

File: lib/chip-types/resource-type.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ var ResourceType = function(parent_module_path, declaration){
 	}
 	Chip.call(this, parent_module_path, true, "resource_type", declaration.name);
 	this.name = declaration.name;
+	this.human_readable_name = declaration.human_readable_name;
+	this.human_readable_description = declaration.human_readable_description;
```
Comment (groovy354) [inline lib/chip-types/resource-type.js]: "Summary" instead of "human_readable_description"

File: lib/chip-types/resource-type.js [hunk 2]
```diff
@@ -157,7 +159,14 @@ ResourceType.prototype = new function(){
 			var field_signature = this.fields[field_name].get_signature(with_validators);
 			resource_type_signature.push(field_signature);
 		}
-		return resource_type_signature;
+		
+		var signature = {
+			name: this.name,
+			human_readable_name: this.human_readable_name,
+			human_readable_description: this.human_readable_description,
+			body: resource_type_signature
```
Comment (groovy354) [inline lib/chip-types/resource-type.js]: "fields" instead of "body"

**Comment by groovy354** [comment]
"fields" should be a hashmap (field_name->field_description)

**Comment by debian-sh** [comment]
I added changes.

```json
[
	{
        "name": "user",
        "human_readable_name": "User",
        "summary": "This resource type was created for handling users, it allows users to set up accounts. After creating an account, the user will be able to login to the your application.",
        "fields": {
            "username": {
                "name": "username",
                "type": "text",
                "required": true,
                "params": {}
            },
            "email": {
                "name": "email",
                "type": "email",
                "required": false,
                "params": {}
            },
            "password": {
                "name": "password",
                "type": "text",
                "required": true,
                "params": {}
            },
            "status": {
                "name": "status",
                "type": "text",
                "required": false,
                "params": {}
            }
        }
    }
]
```

**Comment by debian-sh** [comment]
I must add required information to documentation.

**Pull request closed without merging**